docs/my-first-transaction.md

```diff
@@ -143,7 +146,7 @@ Sample output on success:
 
 ```plaintext
 >> Creating/retrieving next account from wallet
-Created/retrieved account #0 address 3ed8e5fafae4147b2a105a0be2f81972883441cfaaadf93fc0868e7a0253c4a8
+Created/retrieved account #0 address cc2219df031a68115fad9aee98e051e9
```

Comment:
Could also think about just having a placeholder address field here...

```
<SOME_ADDRESS>
```

Unless we think the address format is pretty stable now?

Reply:
> Unless we think the address format is pretty stable now?

The address format should be stable from now on.